dicomIndex.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>NCG</title>
+</head>
+
+<body>
+  <div id="app"></div>
+  <script type="text/javascript" src="/bundle.js"></script>
+  </link>
+  <script src="https://unpkg.com/@ohif/viewer@3.7.4/dist/index.umd.js" crossorigin></script>
+  <script>
+    var containerId = "root";
+    var componentRenderedOrUpdatedCallback = function () {
+      console.log('OHIF Viewer rendered/updated');
+    }
+
+    window.config = {
+      showStudyList: false,
+      servers: {
+        dicomWeb: [
+          {
+            name: 'Orthanc',
+            wadoUriRoot: 'wado',
+            qidoRoot: `${window.location.origin}/dicom-web`,
```

Comment:
It assumes that the HIU application is available in the location origin (host only). There are exceptions to that. Take for example, our AWS, where the application is available only under the "/hiu" context. I think you will need to atleast pass the "context" via a property or configuration. if such a configuration is blank, then it should default to what you are compositing. `${window.location.origin}/{context_path}`